クエリと結果の出典をいくつか抜粋し検証してみます。

In [1]:
import sys
from pathlib import Path
sys.path.append(str(Path("..").resolve()))
from utilities.references import (
    get_source_references,
    get_entity_references, 
    get_community_report_references, 
    get_community_report_references_without_text_units, 
    get_relationship_references, 
    get_relationship_references_without_text_units, 
    get_network_by_name,
    pretty_json_for_notebook
)

# Query 1

織田信長と徳川家康は、豊臣秀吉を介してどのようにつながっていますか？関係の流れを箇条書きで示してください。日本語で回答してください。

## Answer 抜粋 1

- 清洲同盟を起点に信長と家康が直接つながる
  - 信長と家康は初期の協調関係を清洲同盟として結び、近畿を軸とした勢力基盤を築く。これが秀吉介在以前の権力ネットワークの核となる流れである [Data: Reports (35, 264, 27, 203, 256, +more)]。この点は、後の豊臣秀吉介在へつながる出発点として位置づけられる will.
  - 補足情報として、KIYOSU ALLIANCEの性格と機能も同様の結びつきを示す資料がある [Data: Reports (27, 0, 35)]。

In [2]:
ret = get_community_report_references(35)
print(pretty_json_for_notebook(ret))

[
  {
    "community_network_name": "Nobunaga Campaign Network (Nobunaga, Inabayama, Nagashima)",
    "community_report_summary": "The community centers on Oda Nobunaga and his campaigns during the Sengoku period, demonstrating a highly interconnected network of provinces, castles, temples, and regional powers. Nobunaga functions as the hub, linking the Kiyosu Alliance with Tokugawa Ieyasu, and coordinating military moves across Kinai, Mino, and Ise while establishing power centers such as Inabayama Castle. The Nagashima region and surrounding Ise theatre provide ordinary fronts of conflict, with notable battles and sieges shaping the rise and consolidation of Nobunaga’s regime. Religious and cultural sites—such as Hokke-ji in Suwa and Myokaku-ji—appear in the narrative as strategic-context components of campaigns and episodes like the Honno-ji Incident, which precipitated a dramatic reshuffle of power. The story is further reinforced by modern literary and media portrayals, which cura

### 上記文書の400文字要約 (GPT-5.2)

織田信長を中心に、清洲同盟で徳川家康と結びつきながら、畿内・美濃・伊勢など各地の国衆、城郭（稲葉山城など）、寺社を横断して展開する戦役ネットワークを描いています。長島一向一揆や各地の包囲戦・合戦が前線を形成し、軍事行動と領国支配の拠点づくりが同時に進む構図です。寺社（法華寺・妙覚寺など）は宗教施設としてだけでなく、宿所や政治的・戦略的コンテキストとして物語に組み込まれます。最終的に本能寺の変が急激な権力再編を引き起こし、関係（Relationships）と実体（Entities）、史料参照に基づく「つながり」と「根拠」の束として全体像が補強されています。さらに近現代の文学・メディア表象が信長像を更新し続ける点も含意されています。

## Answer 抜粋 2

- 秀吉と家康の結びつき（婚姻・人質外交を通じた同盟）
  - 秀吉は家康（徳川家の基盤を築く人物）と婚姻・人質外交を通じて dynastic の結びつきを作りました。特に秀吉の妹・朝日姫を介した婚姻関係の形成が明記されており、両勢力の間に公式な連携の道を開いたとされます。 [Data: Entities (14)]

In [3]:
ret = get_entity_references(14)
print(pretty_json_for_notebook(ret))

[
  {
    "entity": "TOYOTOMI HIDEYOSHI",
    "document": "徳川家康_wiki.txt",
    "text_unit_id": "1429ac36404ab6516fa879407e50e8fb318a866bf8a68d35bddbcb179e2728d2ae5569877963e614eb1448f0f4d1ffbadccd9cc750ee0c5f3482e75966f36229",
    "text": "# This text is based on the Wikipedia article \"徳川家康\"\n# (https://ja.wikipedia.org/wiki/徳川家康),\n# retrieved via the MediaWiki API.\n# query: https://ja.wikipedia.org/w/api.php?action=query&prop=extracts&titles=%E5%BE%B3%E5%B7%9D%E5%AE%B6%E5%BA%B7&format=json\n#\n# © Wikipedia contributors  \n# Licensed under CC BY-SA 4.0\n\n{\n  \"batchcomplete\": \"\",\n  \"warnings\": {\n    \"extracts\": {\n      \"*\": \"HTML may be malformed and/or unbalanced and may omit inline images. Use at your own risk. Known problems are listed at https://www.mediawiki.org/wiki/Special:MyLanguage/Extension:TextExtracts#Caveats.\"\n    }\n  },\n  \"query\": {\n    \"pages\": {\n      \"5144244\": {\n        \"pageid\": 5144244,\n        \"ns\": 0,\n        \"title\": \"徳川家

### 上記文書の400文字要約 (GPT-5.2)
観点：秀吉と家康の結びつきに焦点を当てて要約

徳川家康と豊臣秀吉の関係は、対立から主従、そして政権継承へと段階的に変化しました。両者は天正12年（1584）の小牧・長久手の戦いで直接対峙し、家康は戦術的に優位に立ちましたが、最終的には秀吉が政治的主導権を握ります。その後、秀吉は妹・朝日姫を家康に嫁がせ、母・大政所を送ることで婚姻関係を結び、家康を豊臣政権に組み込みました。家康は臣従を受け入れ、関東への大移封によって最大の大名として遇され、五大老筆頭に列せられます。秀吉没後は、秀頼の後見役として政権中枢に立ち、その権威と地位を基盤に天下掌握へ進みました。両者の結びつきは、武力ではなく政治的調整と制度設計を通じて、日本の支配構造が豊臣から徳川へ移行する要となった関係だと言えます。

## Answer 抜粋 3

- 最終的な継承関係の結晶：江戸幕府へと続く流れ
  - 関ヶ原の戦いを経て、家康が東軍の勝利を収め江戸幕府を開く流れが成立する点は、信長→秀吉→家康という連携の最終的な結びつきを示す重要な局面としてデータに明記されている [Data: Reports (328, +more)]。
  - この連携の完成形は、豊臣政権の枠組みと徳川政権の継承の連続性を示す要素として、Tokugawa Foundations／Tokugawa Scholarly Networkなどの記述群でも位置づけられている [Data: Reports (166, 236, +more)]。

In [5]:
ret = get_network_by_name("Tokugawa Foundations")
print(pretty_json_for_notebook(ret))

[
  {
    "community_reports_id": "d6fc2db464b77bc7fddfba2cf42c60770bddde44380867df66c797c254944360c92b35d29f321f6505d2d7af8b03dc9fa2acdd9b5b403f3bfa4be4dc8119a7d5",
    "community_network_name": "Tokugawa Foundations: Ieyasu, Sunpu Castle, and Suruga Library",
    "community_report_summary": "This community centers on Tokugawa Ieyasu and the infrastructure and networks that enabled the transition from Sengoku-era fragmentation to a centralized Edo-period state. Core entities include Ieyasu himself, the Sunpu Castle base, and the Suruga Library established there, which together provided political stability, intellectual backbone, and administrative continuity. The relationships reveal a pattern of military campaigns and strategic patronage that reinforced authority, as well as cultural and scholarly exchanges that fed governance and education. Notable lineage carries forward through Ieyasu’s descendants, ensuring dynastic continuity, while key sites such as Fushimi and Hamamatsu Castle

### 上記文書の400文字要約 (GPT-5.2)

本コミュニティは、徳川家康を中心に、戦国期の分権的状況から江戸時代の中央集権国家へ移行する過程を支えた基盤とネットワークを描いています。家康自身に加え、駿府城という政治拠点、そこで整備された駿河文庫が中核となり、政治的安定、知的基盤、行政の継続性を同時に実現しました。軍事行動と戦略的な恩顧関係を通じて権威が強化される一方、文化・学術交流が統治や教育に還元される構造が見られます。家康の子孫へと続く系譜は王朝的連続性を確保し、伏見城や浜松城といった拠点が家康の活動範囲の広がりを支えました。結果として、学問に裏打ちされた中央集権的統治モデルと、歴史認識を支える知の蓄積が、長期にわたり持続する政治秩序を形成したことが示されています。


## Answer 抜粋 4

- 本能寺の変（1582年）を契機に、信長の死後の空白を埋める形で、豊臣秀吉が事実上の後継者として台頭します。秀吉は信長の遺志を継ぎ、全国的な統一と諸大名の再編を推し進める中心的な存在となりました。この過程で家康は秀吉の枠組みの中で重要なパートナーとして位置づけられていきます。 [Data: Sources (3, 0, 4, 165, 172)]

In [9]:
ret = get_source_references(3)
print(pretty_json_for_notebook(ret))

{
  "document_name": "徳川家康_wiki.txt",
  "text": "直臣の岡崎城主となるであろう元康と今川氏による間接統治下で希薄化した家臣団との間の主従関係を再確認する性格を持っていた。また、家臣団に対しては家康が諫言を聞かなければ、義父である御一門衆関口親永と今川氏家老朝比奈親徳に訴え出るようにとも記されており、引き続き松平家への関与が継続することも明記されていた。\\n\\u003C/p\\u003E\\n\\u003Ch3 data-mw-anchor=\\\"清洲同盟から三河国平定\\\" data-mw-fallback-anchor=\\\".E6.B8.85.E6.B4.B2.E5.90.8C.E7.9B.9F.E3.81.8B.E3.82.89.E4.B8.89.E6.B2.B3.E5.9B.BD.E5.B9.B3.E5.AE.9A\\\"\\u003E清洲同盟から三河国平定\\u003C/h3\\u003E\\n\\n\\u003Cp\\u003E永禄3年（1560年）5月、桶狭間の戦いで先鋒を任され、大高城の鵜殿長照が城中の兵糧が足りないことを義元に訴えたため、義元から兵糧の補給を命じられた。しかし織田軍は大高城を包囲しており、兵糧を運び込むには包囲を突破する必要があった。そこで5月18日、鷲津砦と丸根砦の間を突破して、小荷駄を城中に送り込み、全軍無事に引上げた。5月19日、丸根の砦を攻め落とし、朝比奈泰朝は鷲津の砦を攻め落とした。\\n\\u003C/p\\u003E\\u003Cp\\u003E5月19日昼頃、今川義元は織田信長に討たれた。織田方の武将の水野信元は、甥の元康のもとへ、浅井道忠を使者として遣わした。同日夕方、道忠は、元康が守っていた大高城に到着し、義元戦死の報を伝えた。織田勢が来襲する前に退却するようとの勧めに対し、元康はいったん物見を出して桶狭間敗戦を確認した。同日夜半に退城。岡崎城内には今川の残兵がいたため、これを避けて翌20日、菩提寺の大樹寺に入った。ほどなくして今川軍は岡崎城を退去。23日、元康は「捨城ならば拾はん」と言って岡崎城に入城した。岡崎城に入る際、大樹寺住職の登誉天室と相談の上、独自の軍事行動をとり、今川からの独立を果たそうとしたとされる。また桶狭間の戦いの直後から、元康は今川・

In [10]:
ret = get_source_references(0)
print(pretty_json_for_notebook(ret))

{
  "document_name": "徳川家康_wiki.txt",
  "text": "# This text is based on the Wikipedia article \"徳川家康\"\n# (https://ja.wikipedia.org/wiki/徳川家康),\n# retrieved via the MediaWiki API.\n# query: https://ja.wikipedia.org/w/api.php?action=query&prop=extracts&titles=%E5%BE%B3%E5%B7%9D%E5%AE%B6%E5%BA%B7&format=json\n#\n# © Wikipedia contributors  \n# Licensed under CC BY-SA 4.0\n\n{\n  \"batchcomplete\": \"\",\n  \"warnings\": {\n    \"extracts\": {\n      \"*\": \"HTML may be malformed and/or unbalanced and may omit inline images. Use at your own risk. Known problems are listed at https://www.mediawiki.org/wiki/Special:MyLanguage/Extension:TextExtracts#Caveats.\"\n    }\n  },\n  \"query\": {\n    \"pages\": {\n      \"5144244\": {\n        \"pageid\": 5144244,\n        \"ns\": 0,\n        \"title\": \"徳川家康\",\n        \"extract\": \"\\u003Cp\\u003E\\u003Cb\\u003E徳川 家康\\u003C/b\\u003E（とくがわ いえやす、旧字体: \\u003Cspan lang=\\\"ja\\\"\\u003E\\u003Cb\\u003E德川 家康\\u003C/b\\u003E\\u003C/span\\u003E）は、戦国時

In [11]:
ret = get_source_references(4)
print(pretty_json_for_notebook(ret))

{
  "document_name": "徳川家康_wiki.txt",
  "text": "�間の戦い直後は三河の今川方をまとめて織田方の侵攻と対峙していた元康が三河への軍事的支援を後回しにして同盟国の武田・北条支援に動く氏真に失望して、援軍を得られないまま織田氏に抵抗を続けるよりも織田氏と結んで独立を図った方が領国維持の上で得策と判断したとする見方もある。この事態は義元の後を継いだ今川氏真には痛恨の事態であり、後々まで「松平蔵人逆心」「三州錯乱」などと記して憤りを見せている。その後も元康は藤波畷の戦いなどに勝利して、西三河の諸城を攻略する。\\n\\u003C/p\\u003E\\u003Cp\\u003E永禄4年（1561年）先に今川氏を見限り織田氏と同盟を結んだ伯父・水野信元の仲介もあって、信長と和睦し、今川氏と断交して信長と同盟を結んだ（\\u003Cb\\u003E清洲同盟\\u003C/b\\u003E）（『史料総覧』巻10）。同年4月西三河で今川氏との戦いが開始された。\\n\\u003C/p\\u003E\\u003Cp\\u003E永禄5年（1562年）には、元康と信長が会って会談し、同盟の確認をして関係を固めている。一方、今川氏真の要請を受けた将軍・足利義輝は松平・今川両氏の和睦を図り、義輝から北条氏康らに対しても和睦の仲介を指示しているが、和睦は実現しなかった。\\n\\u003C/p\\u003E\\u003Cp\\u003E永禄6年（1563年）7月6日、元康は、\\u003Cb\\u003E家康\\u003C/b\\u003Eと名を改めた（「徳川幕府家譜」）。「元」の字は今川義元の偏諱を与えられたものであるため、この改名により、今川氏からの自立を明確にした。\\n\\u003C/p\\u003E\\u003Cp\\u003E「家」の字を選んだ理由は明確ではない。ほぼ同じ時期に今川義元に倣った花押の形を変更している。改名以前の花押が「元」の字を変形させたものである以上、花押の変更は当然のことであったとも言えるが、これも今川氏からの決別を示したことと言える。こうした動きが桜井・大草の両松平家をはじめとする親今川派を刺激して、翌年の一斉蜂起につながったとする見方がある。同年3月には、同盟の証として嫡男・竹千代（信康）と信長の娘・五徳との婚

In [12]:
ret = get_source_references(165)
print(pretty_json_for_notebook(ret))

{
  "document_name": "織田信長_wiki.txt",
  "text": "年には「弾正忠」を名乗るようにもなっており、弾正忠家の継承者候補として信長と争う立場にあった。\\n\\u003C/p\\u003E\\u003Cp\\u003E同年8月に両者は稲生で激突するが、結果は信長の勝利に終わった（稲生の戦い）。信長は、末盛城などに籠もった信勝派を包囲したが、生母・土田御前の仲介により、信勝・勝家らを赦免した。\\n\\u003C/p\\u003E\\u003Cp\\u003E永禄元年（1558年）、信勝が再び謀反を企てる。この時、信勝を見限った柴田勝家からの密告があり、事態を悟った信長は病と称して信勝を清洲城に誘い出し殺害した。\\n\\u003C/p\\u003E\\u003Cp\\u003E同年7月、信長は、同族の犬山城主・織田信清と協力し、尾張上四郡（丹羽郡・葉栗郡・中島郡・春日井郡）の守護代・織田伊勢守家（岩倉織田家）の当主・織田信賢を浮野の戦いにおいて撃破した。そして、翌年には、信賢の本拠地・岩倉城を陥落させた。\\n\\u003C/p\\u003E\\u003Cp\\u003E永禄2年（1559年）2月2日、信長は約500名の軍勢を引き連れて上洛し、室町幕府の13代将軍・足利義輝に謁見した。村岡幹生によれば、この上洛の目的は、新たな尾張の統治者として幕府に認めてもらうことにあったという。しかし、この目的は達成されなかったと考えられる。\\n\\u003C/p\\u003E\\u003Cp\\u003E一方、天野忠幸によれば、この上洛は尾張の問題だけによるものではなく、前年に足利義輝が正親町天皇を擁した三好長慶に対して不利な形で和睦をせざるを得なかったことによって諸大名が拠って立つ足利将軍家を頂点に立つ武家秩序が崩壊する危機感が高まり、その状況を信長自らが確認する意図もあったとされる。\\n\\u003C/p\\u003E\\n\\u003Ch4 data-mw-anchor=\\\"桶狭間の戦い\\\" data-mw-fallback-anchor=\\\".E6.A1.B6.E7.8B.AD.E9.96.93.E3.81.AE.E6.88.A6.E3.81.84\\\"\\u003E桶狭間の戦い\\u003C/h4\\u003

In [13]:
ret = get_source_references(172)
print(pretty_json_for_notebook(ret))

{
  "document_name": "織田信長_wiki.txt",
  "text": "させた。この頃、織田軍は浅井・朝倉連合軍と小競り合いを繰り返していた。以後の戦況は織田軍有利に展開した。\\n10月3日、信玄が甲府を出陣し、信長はそれを知らず、5日付けで信玄に対して武田上杉間での和睦の仲介に骨を折ったとの書状を送った。\\n\\u003C/p\\u003E\\u003Cp\\u003E11月14日、織田方であった岩村城が開城し、武田方に占拠された（岩村城の戦い）。病死した岩村城主・遠山景任の後家・おつやの方（信長の叔母）は、秋山虎繁（信友）と婚姻し、武田方に転じた。また、徳川領においては徳川軍が一言坂の戦いで武田軍に敗退し、さらに遠江国の二俣城が開城・降伏により不利な戦況となる（二俣城の戦い）。これに対して信長は、家康に佐久間信盛・平手汎秀ら3,000人の援軍を送ったが、12月の\\u003Cb\\u003E三方ヶ原の戦い\\u003C/b\\u003Eで織田・徳川連合軍は武田軍に敗退し、汎秀は討死した。\\n信長は11月20日付けで上杉謙信に「信玄の所行、まことに前代未聞の無道といえり、侍の義理を知らず、ただ今は都鄙を顧みざるの私大、是非なき題目にて候」「永き儀絶（義絶）たるべき事もちろんに候」「未来永劫を経候といえども、再びあい通じまじく候」と書状を送っている。。\\nこの恨みが忘れられなかったのか長篠後に藤孝に「信玄入道表裏を構え、旧恩を忘れ、恣の働き候いける」と申し送っている。\\n\\u003C/p\\u003E\\u003Cp\\u003E同年の12月から翌年正月のあいだのいずれかの時点で、信長は足利義昭に対して17条からなる異見書を送ったと考えられ、詰問文により信長と義昭の関係は悪化している。この異見書は、従来、『永禄以来年代記』の元亀三年九月条の記述から、元亀3年9月に発給されたものだと考えられてきた。しかし、柴裕之によれば、他の複数の史料の記載や前後の事情から、異見書が元亀3年9月に発給されたとは考え難い。柴は、同年12月の三方ヶ原の戦いの敗戦によって、義昭が従来の信長との協調路線に不安を覚えはじめたと述べる。そして、そのことに対する牽制として、この異見書が出されたものであるとする。\\n\\u003C/p\\u003E\\u003

# Query 2

織田信長・豊臣秀吉・徳川家康の三者を、「統一へのプロセス」の観点で比較し、それぞれの役割を1行ずつでまとめてください。日本語で回答してください。

## Answer 抜粋 1

- 秀吉: 秀吉は信長の死後、山崎の戦い・小牧・長久手の戦いを通じて勢力を再編成・拡大し、豊臣政権を樹立して天下統一の基盤を完成させた will [Data: Reports (256, 149, 74, 1306, 1315, +more)]

In [60]:
ret = get_community_report_references(256)
print(pretty_json_for_notebook(ret))

[
  {
    "community_network_name": "Toyotomi Hideyoshi Circle (Yaemon and Allies)",
    "community_report_summary": "This report centers on Toyotomi Hideyoshi and a network of related figures, sites, and events from the Sengoku period. Core to the community is Hideyoshi (162), whose lineage is traced to Kinoshita Yaemon (1227) as a potential father and NAKA (1232) as a mother, establishing a dynastic context through which power and legitimacy circulated. Key allied and rival actors include Nobukatsu (183), who allied with Hideyoshi (186), Ankokuji Eicho (1639), who secretly coordinated with Hideyoshi (1671), and a set of pivotal military campaigns such as Yamazaki (1221) and Komaki-Nagakute (191). The community also intersects with religious and cultural institutions—Komyoji Temple (1235; Hideyoshi entered at age eight via relationship 1214), Todaiji Temple (422) with related development plans (459)—and religious-ritual dimensions involving the Toyokuni Shrine cult (1708, 1709; 1776, 

In [61]:
ret = get_relationship_references(1306)
print(pretty_json_for_notebook(ret))

[
  {
    "relationship_source": "GAMOU UJISATO",
    "relationship_target": "HOSOKAWA TADAOKI",
    "weight": 4.0,
    "document": "豊臣秀吉_wiki.txt",
    "text_unit_id": "bb110871614658f68a52df168e6ea13682b882f4df6febd120b23a8341c64d68914eb93231fb18f75739d8a0fef914df658035ff22cc62096c085e47fcc60041",
    "text": "と誼を結んでいったため、柴田勝家との対立が激しくなった。\\n\\u003C/p\\u003E\\u003Cp\\u003E天正10年（1582年）10月、勝家は滝川一益や織田信孝と共に秀吉に対する弾劾状を諸大名にばらまいた。\\n10月15日、秀吉は養子の羽柴秀勝（信長の四男）を喪主として、信長の葬儀を行う。同年10月20日付堀秀政宛の秀吉書状の宛名には、羽柴の名字が使用されており、すでに秀吉による織田家臣の掌握が始まっていることが分かる。\\n10月28日、秀吉と丹羽長秀、池田恒興は三法師を織田家当主として擁立した清洲会議の決定事項を反故にし、信雄を織田家当主として擁立し主従関係を結ぶ。ただし、これは三法師が成人するまでの暫定的なものであった。\\n\\u003C/p\\u003E\\u003Cp\\u003E12月、越前国の勝家が雪で動けないのを好機と見た秀吉は、信孝が三法師を安土に戻さないことなどを大義名分とし、信孝打倒の兵を挙げる。\\n\\u003C/p\\u003E\\u003Cp\\u003E12月9日、秀吉は池田恒興ら諸大名に動員令を発動し、自ら5万の大軍の指揮を執り宝寺城から出陣し、12月11日に堀秀政の佐和山城に入り、柴田勝家の養子・柴田勝豊が守る長浜城を包囲した。元々勝豊は勝家、そして同じく養子であった柴田勝政らと不仲であった上に病床に臥していたため、秀吉の調略に応じて降伏し、秀吉は長浜城を獲得した。\\n12月16日には美濃国に侵攻し、稲葉一鉄らの降伏や織田信雄軍の合流などもあってさらに兵力を増強した秀吉は、信孝の家老・斎藤

In [62]:
ret = get_relationship_references(1315)
print(pretty_json_for_notebook(ret))

[
  {
    "relationship_source": "MAEDA TOSHIYE",
    "relationship_target": "SHIZUGATAKE BATTLE",
    "weight": 7.0,
    "document": "豊臣秀吉_wiki.txt",
    "text_unit_id": "dd269e06aec03dc78bd1a07a08f5a342986fbf601552a4483deb241d6d56ab73214d712d6739a563cbf16f2fca613b7affbba4f32689ef5c19bb2bc988646dd7",
    "text": "孝の人質を処刑した。はじめは勝家方が優勢であった。\\n\\u003C/p\\u003E\\u003Cp\\u003E4月20日早朝、勝家の重臣・佐久間盛政は、秀吉が織田信孝を討伐するために美濃国に赴いた隙を突いて、奇襲を実行した。この奇襲は成功し、大岩山砦の中川清秀は敗死し、岩崎山砦の高山重友は敗走した。しかしその後、盛政は勝家の命令に逆らってこの砦で対陣を続けたため、4月21日に中国大返しと同様に迅速に引き返してきた秀吉の反撃にあい、さらに前田利家らの戦線離脱もあって柴田軍は大敗を喫し（賤ケ岳の戦い）、柴田勝家は越前に撤退した（美濃大返し）。\\n\\u003C/p\\u003E\\u003Cp\\u003E4月24日、勝家は正室・お市の方と共に自害した。秀吉はさらに加賀国、能登国、越中国も平定し、前田利家には元々の領地である能登国に加えて加賀国のうちの2郡を与え、佐々成政には越中国の支配をこれまで通り安堵した。5月2日（異説あり）には、織田信孝も自害に追い込み、やがて滝川一益も降伏した。\\n\\u003C/p\\u003E\\u003Cp\\u003Eこうして織田家の実力者たちを葬ったことにより、秀吉は家臣第一の地位を確立。表面上は三法師を奉りつつ、実質的に織田家中を差配することになった。\\n\\u003C/p\\u003E\\n\\u003Ch3 data-mw-anchor=\\\"徳川家康との対立と朝廷への接近\\\" data-mw-fallback-anchor=\\\".E5.BE.B3.E5.B7.9D.E5.AE.B6.

## Answer 抜粋 2

導入
織田信長・豊臣秀吉・徳川家康の三者は、戦国時代の「統一へのプロセス」において、それぞれ異なる局面で役割を果たしました。以下は、統一への道筋という観点から見た要約です。

- 信長（織田信長）: 統一へのプロセスの起点をつくり、尾張・美濃の統合と天下布武の印を用いた中央集権の基盤を築いた創始的主導者。[Data: Entities (5, 150, 70, 215); Data: Relationships (1)]
- 秀吉（豊臣秀吉）: 信長の死後、国内統一を完成させる中核として京都を拠点に中央集権を確立し、外征と国内制度整備を通じて統一を実現した実務的統一者。[Data: Entities (162, 14, 55, 219, 229)]
- 家康（徳川家康）: 江戸幕府を開き長期的な統一秩序を確立、戦国時代の終結と安定した統治体制を築いた最終的な統一者。[Data: Entities (1072, 127)]

In [57]:
ret = get_entity_references(5)
print(pretty_json_for_notebook(ret))

[
  {
    "entity": "ODA NOBUNAGA",
    "document": "徳川家康_wiki.txt",
    "text_unit_id": "1429ac36404ab6516fa879407e50e8fb318a866bf8a68d35bddbcb179e2728d2ae5569877963e614eb1448f0f4d1ffbadccd9cc750ee0c5f3482e75966f36229",
    "text": "# This text is based on the Wikipedia article \"徳川家康\"\n# (https://ja.wikipedia.org/wiki/徳川家康),\n# retrieved via the MediaWiki API.\n# query: https://ja.wikipedia.org/w/api.php?action=query&prop=extracts&titles=%E5%BE%B3%E5%B7%9D%E5%AE%B6%E5%BA%B7&format=json\n#\n# © Wikipedia contributors  \n# Licensed under CC BY-SA 4.0\n\n{\n  \"batchcomplete\": \"\",\n  \"warnings\": {\n    \"extracts\": {\n      \"*\": \"HTML may be malformed and/or unbalanced and may omit inline images. Use at your own risk. Known problems are listed at https://www.mediawiki.org/wiki/Special:MyLanguage/Extension:TextExtracts#Caveats.\"\n    }\n  },\n  \"query\": {\n    \"pages\": {\n      \"5144244\": {\n        \"pageid\": 5144244,\n        \"ns\": 0,\n        \"title\": \"徳川家康\",\n

In [58]:
ret = get_relationship_references(1)
print(pretty_json_for_notebook(ret))

[
  {
    "relationship_source": "ODA NOBUNAGA",
    "relationship_target": "KIYOSU ALLIANCE",
    "weight": 9.0,
    "document": "徳川家康_wiki.txt",
    "text_unit_id": "1429ac36404ab6516fa879407e50e8fb318a866bf8a68d35bddbcb179e2728d2ae5569877963e614eb1448f0f4d1ffbadccd9cc750ee0c5f3482e75966f36229",
    "text": "# This text is based on the Wikipedia article \"徳川家康\"\n# (https://ja.wikipedia.org/wiki/徳川家康),\n# retrieved via the MediaWiki API.\n# query: https://ja.wikipedia.org/w/api.php?action=query&prop=extracts&titles=%E5%BE%B3%E5%B7%9D%E5%AE%B6%E5%BA%B7&format=json\n#\n# © Wikipedia contributors  \n# Licensed under CC BY-SA 4.0\n\n{\n  \"batchcomplete\": \"\",\n  \"warnings\": {\n    \"extracts\": {\n      \"*\": \"HTML may be malformed and/or unbalanced and may omit inline images. Use at your own risk. Known problems are listed at https://www.mediawiki.org/wiki/Special:MyLanguage/Extension:TextExtracts#Caveats.\"\n    }\n  },\n  \"query\": {\n    \"pages\": {\n      \"5144244\": {\n  

In [59]:
ret = get_entity_references(1072)
print(pretty_json_for_notebook(ret))

[
  {
    "entity": "徳川家康",
    "document": "徳川家康_wiki.txt",
    "text_unit_id": "0c3e2b875f17256af518faffbfe0e2ba77f993a227c052b4ce148b05faf059b1b51de78f8f042d32c89b7f26de7e5264379d442b1248c4371f589da630884f32",
    "text": "教育委員会 編『市原市史』 中巻、市原市、1986年3月20日。NDLJP:9643783。\\u003C/cite\\u003E\\u003Cspan title=\\\"ctx_ver=Z39.88-2004&rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&rft.genre=book&rft.btitle=%E5%B8%82%E5%8E%9F%E5%B8%82%E5%8F%B2&rft.date=1986-03-20&rft.pub=%5B%5B%E5%B8%82%E5%8E%9F%E5%B8%82%5D%5D&rfr_id=info:sid/ja.wikipedia.org:%E5%BE%B3%E5%B7%9D%E5%AE%B6%E5%BA%B7\\\"\\u003E\\u003Cspan\\u003E \\u003C/span\\u003E\\u003C/span\\u003E\\u003Cspan\\u003E(\\u003Cspan typeof=\\\"mw:File\\\"\\u003E\\u003Cspan title=\\\"要無料登録\\\"\\u003E\\u003C/span\\u003E\\u003C/span\\u003E要登録)\\u003C/span\\u003E\\u003C/li\\u003E\\n\\u003Cli\\u003E\\u003Ccite style=\\\"font-style:normal\\\" class=\\\"citation\\\" id=\\\"CITEREF黒板1936\\\"\\u003E黒板勝美 編『国史大系』 第五十五巻《公卿補任 第三篇》（新訂増補）、国史大系刊行会、1936年8月30日。NDLJ

# Query 3

織田信長・豊臣秀吉・徳川家康の3人の関係が変化する'転換点'を、年代（できる範囲で）付きで3〜5個挙げ、なぜ転換点なのかを説明してください。日本語で回答してください。

## Answer 抜粋 1

#### 転換点1: 1560年頃—桶狭間の戦いと勢力の飛躍的拡大
- なぜ転換点なのか
  - 織田信長が今後の天下統一へ向けて一気に勢力基盤を固める契機となった。1560年の桶狭間の戦いで信長は今後の拡張と統一の道を切り開き、信長の軍事的台頭が中央・西国の勢力図を大きく動かす転機となった。
  - この戦いとその後の動きは、信長の権力基盤づくりと連携関係の構築（後の豊臣・徳川との関係性にも影響を与える）を提示している。
- 関連データの例
  - 1560年の戦いを含む記述: OKEHAZAMA BATTLE（データID 12）および信長の業績・行動を示すノブナガの項目（データID 5）
  - 関連する網の目（信長の拡張・戦略に関する記述）: Relationships ID 2102, 2113, 2103, 2114, 2343 など
- 参考データ
  - [Data: Entities (12, 5, 77); Relationships (2102, 2113, 2103, 2114, 2343)]

In [31]:
ret = get_relationship_references_without_text_units(2102)
print(pretty_json_for_notebook(ret))

{
  "relationship_source": "NOBUNAGA",
  "relationship_target": "ENRYAKU-JI",
  "description": "Nobunaga ordered the destruction of Enryaku-ji as part of subduing rival powers",
  "weight": 9.0
}


In [32]:
ret = get_relationship_references_without_text_units(2113)
print(pretty_json_for_notebook(ret))

{
  "relationship_source": "AZUCHI CASTLE",
  "relationship_target": "NOBUNAGA",
  "description": "Azuchi Castle, built by Nobunaga (Oda Nobunaga), served as his central base for unification campaigns and functioned as his political center and base of operations. The fortress provided a strategic headquarters during Nobunaga’s military campaigns aimed at unifying Japan, enabling the coordination of troop movements, diplomacy, and governance. By establishing Azuchi Castle as a political hub, Nobunaga consolidated power, projected authority, and organized resources to sustain operations across his domain. The dual role of fortress and administrative seat reflects Nobunaga’s innovative approach to centralized command, turning a fortified residence into a functional center of his efforts to unify the realm. Thus, Azuchi Castle stood as a key nexus in Nobunaga’s expansion and governance strategy, serving not only as a military base during campaigns but also as the political heart from which

#### 日本語訳

織田信長によって築かれた安土城は、日本統一を目指す彼の諸戦役における中枢拠点であり、政治的中心地かつ作戦の本拠地として機能しました。この城郭は、信長が日本統一を目指して展開した軍事行動において戦略的な司令部の役割を果たし、兵の移動、外交、統治を統合的に調整することを可能にしました。
安土城を政治の中枢として据えることで、信長は権力を集約し、権威を示すとともに、領国内全域での活動を支えるための資源を組織しました。城塞としての機能と行政の中枢という二重の役割は、要塞化された居城を日本統一のための実務的な中核へと転換した、信長の中央集権的指揮に対する革新的な発想を反映しています。
このように安土城は、信長の拡張と統治戦略における重要な結節点として、軍事遠征時の拠点であると同時に、統一政策を指揮・運営する政治の心臓部としての役割を担っていました。

In [33]:
ret = get_relationship_references_without_text_units(2103)
print(pretty_json_for_notebook(ret))

{
  "relationship_source": "NOBUNAGA",
  "relationship_target": "ISHIYAMA HONGAN-JI",
  "description": "Nobunaga targeted Ishiyama Hongan-ji during the Osaka Hongan-ji War",
  "weight": 9.0
}


In [34]:
ret = get_relationship_references_without_text_units(2114)
print(pretty_json_for_notebook(ret))

{
  "relationship_source": "OSAKA HONGAN-JI WAR",
  "relationship_target": "NOBUNAGA",
  "description": "Nobunaga waged the Osaka Hongan-ji War against Ishiyama Hongan-ji",
  "weight": 9.0
}


In [35]:
ret = get_relationship_references_without_text_units(2343)
print(pretty_json_for_notebook(ret))

{
  "relationship_source": "NAGASHIMA SIEGE",
  "relationship_target": "NOBUNAGA",
  "description": "Nobunaga directed the siege of Nagashima against its defenders",
  "weight": 9.0
}


## Answer 抜粋 2

- 2) 1582年・本能寺の変
  - なぜ転換点か: 信長の急逝により長年の力の空白が生まれ、秀吉が急速に台頭して実権を握る局面へ。三者の力の分布が“信長中心”から“秀吉中心”へ再編され、家康の生存戦略が再配置される契機となりました。
  - 根拠データ: [Data: Reports (3); Data: Sources (119)]

In [36]:
ret = get_community_report_references_without_text_units(119)
print(pretty_json_for_notebook(ret))

{
  "community_network_name": "Hokoji Great Buddha Hall, Gorin Stone Tower, and Uma-zuka",
  "community_report_summary": "This report analyzes three interconnected historic entities tied to Kyoto's Hokō-ji temple complex: Hokō-ji Great Buddha Hall (Daibutsu-den), Gorin Stone Tower, and the Uma-zuka site. The Gorin Stone Tower was relocated to a position behind the Great Buddha Hall, establishing a direct spatial linkage between these two components. The Gorin Stone Tower is presently known as Uma-zuka, which is recognized as the tomb-like mound that is the current name for the site formerly hosting the tower. The entities carry relative prominence scores (3 for Hokō-ji Daibutsu-Den, 2 for Gorin Stone Tower, and 1 for Uma-zuka), suggesting a hierarchical significance within this limited historical triad."
}


#### 日本語訳

本レポートは、京都の方広寺（ほうこうじ）寺院複合体に関連する、相互に結びついた三つの歴史的要素――方広寺大仏殿（Daibutsu-den）、五輪石塔、そして馬塚（Uma-zuka）――を分析するものです。
五輪石塔は大仏殿の背後へと移設され、この二つの構成要素の間に直接的な空間的連関が形成されました。現在、五輪石塔は「馬塚」として知られており、これはかつて石塔が存在していた場所に由来する、墓状の塚に付けられた現在の名称です。
これらの要素には相対的な重要度スコアが付与されており、方広寺大仏殿が3、五輪石塔が2、馬塚が1とされています。この数値は、この限定された歴史的三要素の中における階層的な重要性を示唆しています。

In [37]:
ret = get_source_references(119)
print(pretty_json_for_notebook(ret))

{
  "document_name": "豊臣秀吉_wiki.txt",
  "text": "春・小早川隆景らが高松城の救援に出てきたため、信長に苦境を訴え援軍を要請。ところが当時の毛利氏が高松城救援に用意できた兵力は羽柴軍の半分の15,000ほどでしかなく、救援は不要であった。信長は三職推任問題や皇位継承問題などで朝廷と頻繁に交渉していたため上洛していた。明智光秀はそこを狙って本能寺の変を起こしたが、軍勢を集める理由が問題であった。ところが秀吉の救援要請で援軍に赴くように命じられたため、信長に疑われることなく軍勢を集め、その軍勢で光秀は京都の信長を討ち果たす。光秀が近衛前久と内通していた説があるように、秀吉も大納言の勧修寺晴豊らと内通しており、その筋から光秀の謀反計画を知り、要請を行ったとされる。\\n\\u003C/p\\u003E\\u003Cp\\u003Eまた、秀吉の中国大返しに関しても、沼城から姫路城まで70キロの距離をわずか1日で撤収しており、秀吉が優秀だったとはいえ、事前に用意をしていなければ不可能なこと、中国大返し後の織田方有力武将への切崩しの異常な速さ、変を知らせる使者は本当に毛利方と間違えて秀吉の陣に入ってきたのか、変後の毛利方との迅速な講和は事前に信長が討たれることを見越して秀吉が小早川隆景・安国寺恵瓊などへ根回しを行っていた結果なのか、など疑惑が持たれている。\\n\\u003C/p\\u003E\\u003Cp\\u003E上記の説についての反論には以下のものがある。\\n\\u003C/p\\u003E\\n\\u003Cul\\u003E\\u003Cli\\u003E『信長公記』によれば、高松城への援軍、西国への出陣を立案したのは信長自身であり、秀吉は毛利家主力の出陣を報告したのみで、秀吉側から援軍の要請があったという記述はない。\\u003C/li\\u003E\\n\\u003Cli\\u003E『浅野家文書』には毛利軍5万人と記されており、秀吉は初期情報のこの数字を元に信長の援軍を請求した。\\u003C/li\\u003E\\n\\u003Cli\\u003E秀吉の援軍要請は、手柄を独占することによって信長に疑念を持たれるのを避ける（信長自身を招いて信長に手柄を譲る）ための策であり、有利な状況でありながら援軍を求める必然

# Query 4

もし織田信長が本能寺の変で倒れていなかった場合、豊臣秀吉と徳川家康の立ち位置は本文の記述からどう変わり得るか、確実に言えること／推測に留まることを分けて述べてください。日本語で回答してください。

## Answer 抜粋 1

- 本能寺の変は1582年に発生し、織田信長はこの事件で死去した。
  出典データ: 本能寺関連の事件記述および信長の死の事実は、HONNOJI INCIDENT（166）および HONNOJI NO HEN（154）と Nobunaga（5）に基づく記述で裏付けられます。[Data: Entities (166, 154, 5)]

In [38]:
ret = get_entity_references(166)
print(pretty_json_for_notebook(ret))

[
  {
    "entity": "HONNOJI INCIDENT",
    "document": "徳川家康_wiki.txt",
    "text_unit_id": "926111cfaa4138b1930956a6b071e10e685a1c6d6c2d2b731d2089b783824491ac930db70a220c0cd3ca583cd8e37c8be4c3e92bce95a86d3c0303bf9a4487b7",
    "text": "とともに居城・安土城を訪れ、大接待を受けた。この際、秀吉より援軍要請があった信長は自ら出陣することを決めたが、家康もこれに従い帰国後に軍勢を整えて西国へ出陣する予定だった。\\n\\u003C/p\\u003E\\u003Cp\\u003E6月2日、堺を遊覧中に京で本能寺の変が起こった。このときの家康の供は小姓衆など少人数であったため極めて危険な状態となり、一時は狼狽して信長の後を追おうとするほどであった。しかし本多忠勝に説得されて翻意し、服部半蔵の進言を受け、伊賀国の険しい山道を越え加太越を経て伊勢国から海路で三河国に辛うじて戻った（神君伊賀越え）。帰国後、家康は直ちに兵を率いて上洛しようとしたが、鳴海で秀吉が光秀を討った報を受けて引き返した。穴山信君が帰国途中で戦死したため、駿河国江尻を併呑した。\\n\\u003C/p\\u003E\\u003Cp\\u003E一方、織田氏の領国となっていた旧武田領の甲斐国と信濃国では大量の一揆が起こった。さらに、越後国の上杉氏、相模国の北条氏も旧武田領への侵攻の気配を見せた。旧武田領国のうち上野一国と信濃小県郡・佐久郡の支配を担っていた滝川一益は、旧武田領を治めてまだ3か月ほどしか経っておらず、軍の編成が済んでいなかったことや、武田遺臣による一揆が相次いで勃発したため、滝川配下であった信濃国の森長可と毛利秀頼は領地を捨て畿内へ敗走した。また、甲斐一国と信濃諏訪郡支配を担った河尻秀隆は一揆勢に敗れ戦死するなど緊迫した状況にあった。追い打ちをかけるように、織田氏と同盟関係を築いていた北条氏が一方的に同盟を破り、北条氏直率いる6万の軍が武蔵・上野国境に襲来した。滝川一益は北条氏直を迎撃、緒戦に勝利するも敗北、尾張国まで敗走した。このため、甲斐・信濃・上野は領主のい

In [39]:
ret = get_entity_references(154)
print(pretty_json_for_notebook(ret))

[
  {
    "entity": "HONNOJI NO HEN",
    "document": "徳川家康_wiki.txt",
    "text_unit_id": "f258c9764db6cd7f42bf4a901e13787cb74e681f0417a23bcb558e097a67f5bda2eea9d93841ce06a43e98919d7fcfafe19a86c4aebc71b19a14b1ed29a57274",
    "text": "害と信康の切腹は家康・信康父子の対立が原因とする説や、築山殿や信康は実際に武田氏に内通して家康への謀反を企んだとする説も出されている。なお家康本人は堀秀政宛に「今度左衛門尉(酒井忠次)をもって申し上げ候処、種々御懇ろ之儀、其の段お取りなし故に候。忝き意存に候。よって三郎不覚悟に付いて、去る四日岡崎を追い出し申し候。猶其の趣小栗大六・成瀬藤八(国次)申し入るべきに候。恐々謹言」としている（松平信康#信康自刃事件および家康との関係の項を参照）。嫡男・義信を死なせた宿敵にしてお手本たる信玄と同様の悲劇であった。\\n\\u003C/p\\u003E\\u003Cp\\u003E岩村城の戦い以降に織田氏と武田氏は大規模な抗争をしておらず、後北条氏との対立をも抱えることにもなった勝頼は人質にしていた信長の五男・勝長を返還するなど織田氏との和睦（甲江和与）を模索している。しかし、信長はこれを黙殺し、天正9年（1581年）、降伏・開城を封じた上での総攻撃によって家康は高天神城を奪回する（高天神城の戦い）。高天神城落城、しかも後詰を送らず見殺しにしたことは武田氏の威信を致命的に失墜させ、国人衆は大きく動揺した。木曾義昌の調略成功をきっかけに、天正10年（1582年）2月に信長は家康と共同で武田領へ本格的侵攻を開始した。織田軍の信濃方面からの侵攻に呼応して徳川軍も駿河方面から侵攻し、甲斐南部の河内領・駿河江尻領主の穴山信君（梅雪）を調略によって離反させるなどして駿河領を確保した。勝頼一行は同年3月に天目山で自害して武田氏は滅亡した。最後まで抵抗した武田方の蘆田信蕃（依田信蕃）が守る田中城は成瀬正一らの説得により大久保忠世に引き渡された。\\n\\u003C/p\\u003E\\u003Cp\\u003E家康は3月10日に

In [40]:
ret = get_entity_references(5)
print(pretty_json_for_notebook(ret))

[
  {
    "entity": "ODA NOBUNAGA",
    "document": "徳川家康_wiki.txt",
    "text_unit_id": "1429ac36404ab6516fa879407e50e8fb318a866bf8a68d35bddbcb179e2728d2ae5569877963e614eb1448f0f4d1ffbadccd9cc750ee0c5f3482e75966f36229",
    "text": "# This text is based on the Wikipedia article \"徳川家康\"\n# (https://ja.wikipedia.org/wiki/徳川家康),\n# retrieved via the MediaWiki API.\n# query: https://ja.wikipedia.org/w/api.php?action=query&prop=extracts&titles=%E5%BE%B3%E5%B7%9D%E5%AE%B6%E5%BA%B7&format=json\n#\n# © Wikipedia contributors  \n# Licensed under CC BY-SA 4.0\n\n{\n  \"batchcomplete\": \"\",\n  \"warnings\": {\n    \"extracts\": {\n      \"*\": \"HTML may be malformed and/or unbalanced and may omit inline images. Use at your own risk. Known problems are listed at https://www.mediawiki.org/wiki/Special:MyLanguage/Extension:TextExtracts#Caveats.\"\n    }\n  },\n  \"query\": {\n    \"pages\": {\n      \"5144244\": {\n        \"pageid\": 5144244,\n        \"ns\": 0,\n        \"title\": \"徳川家康\",\n

## Answer 抜粋 2

データ出典の例
- 確実に言えることの根拠としては、信長存命下での京都・中央権力の安定化と后継軸の再定義に関する記述が中心です。[Data: Reports (3); Data: Sources (93)]
- 京都儀礼・権威の投射・拠点の機能に関する点は、京都儀礼・Azuchi Castleの役割に関する記述と整合します。[Data: Reports (3); Data: Entities (151); Data: Relationships (2113)]
- さらに、義昭の abdication軸が存命下では成立しにくいとの見解は、存命前提の分岐論の核として位置づけられています。[Data: Reports (3); Data: Sources (93)]

In [41]:
ret = get_relationship_references(2113)
print(pretty_json_for_notebook(ret))

[
  {
    "relationship_source": "AZUCHI CASTLE",
    "relationship_target": "NOBUNAGA",
    "weight": 20.0,
    "document": "織田信長_wiki.txt",
    "text_unit_id": "2392f6eb689b3feba09d91a28be01b5f4a2cba34338452793deba0032e03ebd9e4cfad10db9ec97db8eaf4d0c7377e3c354b517d1c7c42275479b5292dcc1387",
    "text": "興を志す意を込めて、「天下布武」の印を使用した。\\n\\u003C/p\\u003E\\u003Cp\\u003E翌年10月、足利義昭とともに信長は上洛し、三好三人衆などを撃破して、室町幕府の再興を果たす。信長は、室町幕府との二重政権（連合政権）を築いて、「天下」（五畿内）の静謐を実現することを目指した。しかし、敵対勢力も多く、元亀元年（1570年）6月、越前の朝倉義景・北近江の浅井長政を姉川の戦いで破ることには成功したものの、三好三人衆や比叡山延暦寺、石山本願寺などに追い詰められる。同年末に、信長と義昭は一部の敵対勢力と講和を結び、ようやく窮地を脱した。\\n\\u003C/p\\u003E\\u003Cp\\u003E元亀2年（1571年）9月、比叡山を焼き討ちする。しかし、その後も苦しい情勢は続き、三方ヶ原の戦いで織田・徳川連合軍が武田信玄に敗れた後、元亀4年（1573年）、将軍・足利義昭は信長を見限る。信長は義昭と敵対することとなり、同年中には義昭を京都から追放した（槇島城の戦い）。\\n\\u003C/p\\u003E\\u003Cp\\u003E将軍不在のまま中央政権を維持しなければならなくなった信長は、天下人への道を進み始める。元亀から天正への改元を実現すると、天正元年（1573年）中には浅井長政・朝倉義景・三好義継を攻め、これらの諸勢力を滅ぼすことに成功した。天正3年（1575年）には、長篠の戦いでの武田氏に対して勝利するとともに、右近衛大将に就任し、室町幕府に代わる新政権の構築に乗り出した。翌年には安土城の築城も開始している。しかし、天正5年（

In [42]:
ret = get_community_report_references_without_text_units(3)
print(pretty_json_for_notebook(ret))

{
  "community_network_name": "Nobunaga–Ashikaga Sengoku Network (Azuchi Castle & Major Campaigns)",
  "community_report_summary": "The community centers on two dominant historical actors, Nobunaga (Oda Nobunaga) and Ashikaga Yoshiaki, during the Sengoku period. It integrates military, political, religious, and ceremonial dimensions through pivotal sites (e.g., Azuchi Castle, Ishiyama Hongan-ji), major armed campaigns (Nagashima, Shitaragahara, Enryaku-ji destruction), notable events (Honnoji Incident), and dynastic linkages (Tokuhime). The relationships show Nobunaga driving strategic campaigns and governance, while Yoshiaki’s rise and exile mark shifts in authority, with ceremonial displays and religious-political networks shaping the broader power dynamic. This network is evidenced by a constellation of entities and documented campaigns that collectively illuminate the trajectory toward unification and the era’s disruptive politics [Data: Entities (77, 89, 151, 2268, 1347, 2333, 225

#### 日本語訳

本コミュニティは、戦国時代における二人の支配的な歴史的主体――織田信長と足利義昭――を中心に構成されています。安土城や石山本願寺といった重要拠点、長島・設楽原・比叡山延暦寺焼き討ちなどの主要な軍事行動、本能寺の変といった著名な事件、さらに徳姫に代表される王朝的・血縁的なつながりを通じて、軍事・政治・宗教・儀礼といった複数の次元が統合されています。
これらの関係性を見ると、信長が戦略的な軍事行動と統治を主導する存在として描かれる一方、義昭の擁立と追放は権力構造の転換点を示しており、儀礼的な演出や宗教と政治が交錯するネットワークが、より広範な権力力学を形作っていたことがうかがえます。
このネットワークは、多数の関連エンティティや記録された軍事行動によって裏づけられており、それらが総体として、日本統一へと向かう歴史的軌跡と、当時の破壊的かつ流動的な政治状況を明らかにしています［Data: Entities（77, 89, 151, 2268, 1347, 2333, 2251, 2257, 76, 2096, 2097）；Relationships（2113, 2102, 2103, 2114, 2368, 2466, 2535, 2118, 2343, 2355, 2626, 2697, 2102, 92）］。

# Query 5

織田信長・豊臣秀吉・徳川家康の間にある関係を「同盟」「敵対」「主従（上下関係）」「協力」「継承（政権の引き継ぎ）」に分類し、本文から根拠になりそうな出来事や状況も添えて整理してください。日本語で回答してください。

## Answer 抜粋 1

2) 敵対
- Sekigaharaの戦い（1600年）を境に、東軍（家康）と西軍（豊臣系を含む勢力）との対立が顕在化。戦局は東軍の優位へと傾き、関係性が再編される転換点となりました。
  - 根拠: Sekigahara 関連の概説・戦局記述 [Data: Reports (3); Data: Sources (46)]
- 西軍の大名の寝返り（小早川秀秋の東軍転向）など、戦局を決定づける謀略・転向が戦局を動かしました。
  - 根拠: 小早川秀秋の転向・東軍優勢化の記録 [Data: Sources (23)]
- 大坂の陣（1614–1615年）前後の東西勢力図の最終的な再編と豊臣政権の衰退は、三者関係の力関係を徳川有利へと確定づけました。
  - 根拠: 大坂の陣関連の戦局・講和・陥落の記述 [Data: Reports (3)]
- 総括
  - 三者間での直接的な「敵対関係」が全期間を通じて一貫していたわけではなく、局面ごとに三者の間で協力・対立が変動する形態が多く見られます。Sekigaharaを境に東軍優勢・豊臣政権の衰退が進みました [Data: Reports (3); Data: Sources (46)]。

In [48]:
ret = get_community_report_references(3)
print(pretty_json_for_notebook(ret))

[
  {
    "community_network_name": "Nobunaga–Ashikaga Sengoku Network (Azuchi Castle & Major Campaigns)",
    "community_report_summary": "The community centers on two dominant historical actors, Nobunaga (Oda Nobunaga) and Ashikaga Yoshiaki, during the Sengoku period. It integrates military, political, religious, and ceremonial dimensions through pivotal sites (e.g., Azuchi Castle, Ishiyama Hongan-ji), major armed campaigns (Nagashima, Shitaragahara, Enryaku-ji destruction), notable events (Honnoji Incident), and dynastic linkages (Tokuhime). The relationships show Nobunaga driving strategic campaigns and governance, while Yoshiaki’s rise and exile mark shifts in authority, with ceremonial displays and religious-political networks shaping the broader power dynamic. This network is evidenced by a constellation of entities and documented campaigns that collectively illuminate the trajectory toward unification and the era’s disruptive politics [Data: Entities (77, 89, 151, 2268, 1347, 2

In [44]:
ret = get_source_references(46)
print(pretty_json_for_notebook(ret))

{
  "document_name": "徳川家康_wiki.txt",
  "text": ".81.AE.E5.8F.96.E3.82.8A.E6.89.B1.E3.81.84.E3.81.AB.E9.96.A2.E3.81.97.E3.81.A6\\\"\\u003E一族・譜代の取り扱いに関して\\u003C/h3\\u003E\\n\\u003Cp\\u003E息子や家臣に対しても冷酷非情な面を見せる人物だったとされることが多いが、\\u003Cspan title=\\\"この記述は言葉を濁しているか情報の出所を曖昧にしています。（2023年1月）\\\"\\u003E情に流されず息子や一族に対しても一律に公平であったと見る向きもある。\\u003C/span\\u003E\\n\\u003C/p\\u003E\\u003Cp\\u003E長男・信康の切腹に関しては、信長の要求によるものではなく、家康自らの粛清説も近年唱えられている。また、生母の身分が低い次男・結城秀康、六男・忠輝を、出生の疑惑や容貌が醜いなどの理由で常に遠ざけていたとされるが、これには異論もある。\\n\\u003C/p\\u003E\\u003Cp\\u003E関ヶ原の戦いにおいて江戸留守居役を命じられた秀康は、戦功を挙げるために秀忠に代わり西上したいと申し出たが容れられなかった。かねてから秀康には石田三成との交流があり、豊臣方に内通する恐れがあったとも考えられる一方で、武将として実績のある秀康に三成と友誼が深く西軍に呼応する恐れが強い佐竹義宣を監視させ、東北戦線で上杉氏と戦う伊達政宗・最上義光らの後詰め役として待機させたとされる。秀康は後の論功行賞において破格の50万石を加増、官位も権中納言まで昇進しており、最終的に67万石もの大封を与えられ、江戸への参勤免除、幕府からの使役の免除、関所を大砲で破壊しても黙認されるなど、別格の扱いを受けている。将軍継嗣がならなかったのは、豊臣秀吉の養子で、後に結城家に養子に入り名跡を継いでいることなどが理由とされる。また、秀忠の生母である於愛（竜泉院）は、築山殿の死後に家康の正室とされていた時期があったのではないかとする新説も出されており、この説が正しければ秀忠の後継はかなり早い時期からの既定路線であったとも考えられる。なお、秀康の子・松平忠直には

In [53]:
ret = get_source_references(23)
print(pretty_json_for_notebook(ret))

{
  "document_name": "徳川家康_wiki.txt",
  "text": "軍就任と江戸開府\\\" data-mw-fallback-anchor=\\\".E5.BE.81.E5.A4.B7.E5.A4.A7.E5.B0.86.E8.BB.8D.E5.B0.B1.E4.BB.BB.E3.81.A8.E6.B1.9F.E6.88.B8.E9.96.8B.E5.BA.9C\\\"\\u003E征夷大将軍就任と江戸開府\\u003C/h3\\u003E\\n\\u003Cp\\u003E関ヶ原の戦いの戦後処理を終わらせた慶長6年（1601年）3月23日、家康は大坂城・西の丸を出て伏見城にて政務を執り、征夷大将軍として幕府を開くため、徳川氏の系図の改姓を行った。\\n\\u003C/p\\u003E\\u003Cp\\u003E慶長7年（1602年）、関ヶ原の戦いの戦後処理で唯一処分が決まっていなかった常陸国水戸の佐竹義宣を出羽国久保田に減転封。代わりに佐竹氏と同じく源義光の流れをくむ武田氏を継承した五男・武田信吉を水戸に入れた。これによって確定した徳川氏の領域は一門・譜代大名の所領も含めると、東は岩城領から関東一円、北は南信濃から美濃国・越前国、西は近江国・山城国・大和国と北伊勢の桑名領をほぼ一円支配しつつ、西国では長崎や堺、石見銀山、生野銀山の直轄領が点在する内容であった（秋田氏や里見氏などの小規模な外様大名の支配地は除く）。\\n\\u003C/p\\u003E\\u003Cp\\u003E慶長8年（1603年）2月12日、家康は右大臣に任じられる。同日、征夷大将軍、源氏長者、奨学淳和院等別当、牛車兵仗等の宣下があった。この征夷大将軍就任をもって、一般には「江戸開府」（江戸幕府の創設）とみなされている。\\n\\u003C/p\\u003E\\u003Cp\\u003E同年3月12日、伏見城から二条城に移り、3月21日、衣冠束帯を纏い行列を整えて御所に参内し、将軍拝賀の礼を行い、年頭の祝賀も述べた。3月27日、二条城に勅使を迎え、重臣や公家衆を招いて将軍就任の祝賀の儀を行った。また4月4日から3日間、二条城で能楽が行われ諸大名や公家衆を饗応した。\\n\\u003C/p\\u003E\\u003Cp\\u003Eなお家康の将軍宣下の数ヵ月前の

## Answer 抜粋 2

1) 同盟
- 永禄5年頃（1562年）に信長と徳川家康が清洲同盟を結び、同盟の確認が行われたとされる。これは両者の実務的協力関係の出発点とされます。
  - 根拠: 永禄5年1562年頃の清洲同盟成立・会談確認の記述 [Data: Sources (3)]
- 清洲同盟を背景に、信長と家康は長期的な協力関係を維持・活用する枠組みを作り、以後の勢力配置に影響を与えました。
  - 根拠: Nobunaga–Ashikaga Sengoku Network の「KIYOSU ALLIANCE」関連エビデンス [Data: Nobunaga–Ashikaga Sengoku Network (151); Nobunaga–Ashikaga Sengoku Network (2113)]
- 婚姻外交などの dynastic ties を通じた協力関係の深化も同盟の補強となりました。
  - 根拠: 朝日姫の婚姻（1586年頃）と関連する記録 [Data: Reports (3); Data: Sources (12)]
  - 併せて、信長の娘 Tokuhime の婚姻関係を通じた家臣・勢力間の結びつきの強化を示すエントリも参照可能です
    - 根拠: Tokuhime（信長の娘）関連エンティティ/関係 [Data: Entities (76); Data: Relationships (92)]
- 京都儀礼の権威表現も、同盟関係の正統性・協力の雰囲気を後押ししました。
  - 根拠: 京都儀礼・儀式的行動の記述（儀礼が同盟関係の正統性演出に寄与） [Data: Reports (3); Data: Entities (2333); Data: Relationships (2118)]
- 摂関・公家・寺社勢力を含む儀礼的協力の枠組みと、信長・秀吉・家康の連携の文脈を示す一次資料類
  - 根拠: 信長政権期の儀礼・宮廷関係の記述群 [Data: Sources (184, 153, 206)] ほか、京都儀礼の文脈 [Data: Reports (3)]
- 総括
  - 同盟の性質は初期には二者間の協力・同盟として成立・維持され、その土台のうえで dynastic ties や儀礼表現が協力関係を補強しました [Data: Sources (3); Data: Entities (151); Data: Relationships (2113); Data: Entities (76); Data: Relationships (92)].

In [54]:
ret = get_relationship_references_without_text_units(2113)
print(pretty_json_for_notebook(ret))

{
  "relationship_source": "AZUCHI CASTLE",
  "relationship_target": "NOBUNAGA",
  "description": "Azuchi Castle, built by Nobunaga (Oda Nobunaga), served as his central base for unification campaigns and functioned as his political center and base of operations. The fortress provided a strategic headquarters during Nobunaga’s military campaigns aimed at unifying Japan, enabling the coordination of troop movements, diplomacy, and governance. By establishing Azuchi Castle as a political hub, Nobunaga consolidated power, projected authority, and organized resources to sustain operations across his domain. The dual role of fortress and administrative seat reflects Nobunaga’s innovative approach to centralized command, turning a fortified residence into a functional center of his efforts to unify the realm. Thus, Azuchi Castle stood as a key nexus in Nobunaga’s expansion and governance strategy, serving not only as a military base during campaigns but also as the political heart from which

#### 日本語訳

織田信長によって築かれた安土城は、日本統一を目指す諸戦役における中核拠点として機能すると同時に、彼の政治的中心地であり作戦遂行の本拠でもありました。この城郭は、日本統一を目的とした信長の軍事行動において戦略的な司令部の役割を果たし、兵力の移動、外交、統治を一体的に調整することを可能にしました。
安土城を政治の中枢として位置づけることで、信長は権力を集約し、権威を誇示するとともに、領国全体にわたる作戦を継続するための資源を組織しました。要塞としての機能と行政の中枢という二重の役割は、防御的な居城を、日本統一事業のための実務的な拠点へと転換した、信長の中央集権的統治に対する革新的な発想を反映しています。
このように安土城は、信長の拡張および統治戦略における重要な結節点として、軍事遠征時の拠点であるだけでなく、日本統一構想を指揮・運営する政治の中心としての役割を担っていました。

In [55]:
ret = get_relationship_references_without_text_units(92)
print(pretty_json_for_notebook(ret))

{
  "relationship_source": "NOBUNAGA",
  "relationship_target": "TOKUHIME",
  "description": "Tokuhime is Nobunaga's daughter",
  "weight": 18.0
}


In [56]:
ret = get_relationship_references_without_text_units(2118)
print(pretty_json_for_notebook(ret))

{
  "relationship_source": "NOBUNAGA",
  "relationship_target": "KYOTO HORSE PARADE",
  "description": "Nobunaga organized and commanded the Kyoto Horse Parade as a ceremonial and military display in Kyoto, using the event to project his power and authority in the city. The parade served as a public demonstration of strength and governance, aligning ceremonial spectacle with political signaling. Additionally, a Nobunaga quotation about kingship is described as occurring just before the Kyoto Horse Parade, linking Nobunaga to the event and framing the occasion within his discourse on rulership. Taken together, the descriptions portray Nobunaga as the organizer and leader of the Kyoto Horse Parade, leveraging the ceremonial procession to display power in Kyoto and tying his leadership to both the spectacle and his rhetoric on kingship.",
  "weight": 16.0
}


#### 日本語訳

織田信長は、京都において行われた「上洛馬揃え（京都馬揃え）」を、儀礼的かつ軍事的な示威行動として組織・指揮し、この行事を通じて都における自らの権力と権威を誇示しました。この馬揃えは、武力と統治力を公に示す場として機能し、壮麗な儀礼的演出と政治的なメッセージを結びつける役割を果たしていました。
さらに、王権に関する信長の言葉が、この京都馬揃えの直前に語られたものとして描写されており、信長自身の統治観・君主観とこの行事とを結びつけ、出来事全体を彼の支配論の文脈の中に位置づけています。
これらを総合すると、信長は京都馬揃えの主催者かつ指導者として描かれており、この儀礼的行列を用いて京都における権力を可視化すると同時に、壮観な見世物と王権をめぐる自身の言説の双方を通じて、自らの指導性を強く印象づけていたことが分かります。